docs/setup_ovos_services.md

```diff
@@ -0,0 +1,191 @@
+# Running & Setting Up OVOS Services
+
+OpenVoiceOS is a software stack that includes seven important services, with some being optional, depending on the platform or environment. If you are a developer or would like to quickly test OpenVoiceOS for debugging purposes, you can run each service separately in a terminal. However, for users, distributions, and packagers, it is recommended to use systemd or any other init service that their userland supports for launching and managing OpenVoiceOS services.
+
+### Here are the commands for launching each of the services from the CLI:
+
+ - Launching the Messagebus service (required): python3 -m mycroft.messagebus.service
+ - Launching the Skills service (required): python3 -m mycroft.skills
+ - Launching the PHAL service (required): python3 -m ovos_PHAL
+ - Launching the Audio service (required): python3 -m mycroft.audio
+ - Launching the Voice service (required): python3 -m mycroft.client.speech
+ - Launching the GUI service (optional): python3 -m mycroft.gui
+ - Launching the CLI service (optional): python3 -m mycroft.client.text
+
+OpenVoiceOS services can be set up as SYSTEM or USER systemd services. The choice of which type to use and how to set them up is left up to the user, packager, and distributions. Keep in mind that these service scripts are just examples and may require changes based on your requirements.
+
+### Setup the OpenVoiceOS service:
+- Create a service file called "ovos.service" under "/usr/lib/systemd/user/" with the following contents:
+
+```
+[Unit]
+Description=OpenVoiceOS Software stack.
+Requires=ovos-messagebus.service
+Requires=ovos-skills.service
+Requires=ovos-audio.service
+Requires=ovos-voice.service
+Requires=ovos-gui.service
+Requires=ovos-phal.service
+
+[Service]
+Type=simple
+ExecStart=/bin/true
+RemainAfterExit=yes
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Here are the service files required for each of the individual services:
+
+### Setting up Messagebus service as a user service:
+- Create a service file called "ovos-messagebus.service" under "/usr/lib/systemd/user/" with the following contents:
+
+```
+[Unit]
+Description=OpenVoiceOS Messagebus
+Requires=ovos.service
+PartOf=ovos.service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/python -m mycroft.messagebus.service
+TimeoutStartSec=1m
+TimeoutStopSec=1m
+Restart=on-failure
+StartLimitInterval=5min
+StartLimitBurst=4
+StandardOutput=append:/var/log/ovos/bus.log
```

Comment:
I would not keep these here either, ovos is logging to XDG paths already, this is duplicating logs